clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py

```diff
@@ -114,6 +141,10 @@ def cli(
     stride_size: int = 50,
     slice_direction: int = 0,
     slice_mode: str = "rgb",
+    roi_list: Optional[List[str]] = None,
+    roi_uncrop_output: bool = False,
+    custom_template: Optional[str] = None,
+    custom_mask_pattern: str = "",
```

Comment:
there seems to be an issue with this part of the code (instantiation fails) with message:
AttributeError: 'tuple' object has no attribute 'expandtabs' (c.f console log in CI)

Reply:
Misplaced comma, fixed.